qa/rolling-upgrade/src/test/java/org/opensearch/upgrades/IndexingIT.java

```diff
@@ -267,7 +268,7 @@ public void testIndexingWithSegRep() throws Exception {
                     .put(IndexMetadata.SETTING_REPLICATION_TYPE, ReplicationType.SEGMENT)
                     .put(
                         EngineConfig.INDEX_CODEC_SETTING.getKey(),
-                        randomFrom(CodecService.DEFAULT_CODEC, CodecService.BEST_COMPRESSION_CODEC, CodecService.LUCENE_DEFAULT_CODEC)
+                        randomFrom(OpenSearchIntegTestCase.CODECS)
```

Comment:
Should we do `OpenSearchIntegTestCase.CODECS + CodecService.LUCENE_DEFAULT_CODEC` here as well?  (and in `SegmentReplicationIT`)

Reply:
we can do that, I'll update!

Reply:
@reta updated!!